docs/metasploit-framework.wiki/How-to-Configure-DNS.md

```diff
@@ -0,0 +1,134 @@
+# Metasploit DNS
+## Background
+Most applications that need to handle hostname to IP address lookups rely on the host operating system, either by
+passing the hostname directly to the socket-creation function or by calling a purpose built API such as `getaddrinfo`.
+This was also how Metasploit handled name lookups and would only directly communicate with a DNS server when the request
+was more involved than mapping a hostname to an IPv4 or IPv6 address.
+
+One flaw in this approach is that when pivoting connections over a session, the DNS lookups would occur through the host
+on which Metasploit was running instead of the compromised host from which the connection would originate. This lead to
+two issues, the first being the aforementioned DNS leaks and the second that Metasploit could not always resolve
+hostnames that the compromised system could.
+
+Starting in Metasploit 6.4, Metasploit uses an internal DNS resolution system that grants the user a high degree of
+control over the process of DNS queries.
+
+## The DNS command
+Metasploit's DNS configuration is controlled by the `dns` command which has multiple subcommands. This command is only
+available when the `dns_feature` is enabled (`set feature dns_feature true`). Once enabled, the current configuration
```

Comment:
It's implied here but maybe explicitly mention that the `dns_feature` by default is disabled. 
Also does the `dns_feature` require a database to be connected to metasploit? If so it might be worth mentioning that here.